src/Compiler/Microsoft.CodeAnalysis.Razor/src/ComponentTagHelperDescriptorProvider.cs

```diff
@@ -530,28 +521,28 @@ private IEnumerable<(IPropertySymbol property, PropertyKind kind)> GetProperties
                     kind = PropertyKind.Enum;
                 }
 
-                if (kind == PropertyKind.Default && SymbolEqualityComparer.Default.Equals(property.Type, symbols.RenderFragment))
+                if (kind == PropertyKind.Default && property.Type.HasFullName(ComponentsApi.RenderFragment.MetadataName))
```

Comment:
Can't `property.Type` be `null`?

Reply:
Yeah, it can, although rare in practice. The code directly above (unchanged) already directly dereferences it, so this would blow up regardless of my change. Given that I'd rather not try and fix it in this PR, but we probably want to do a few nullability passes on this code anyway.